examples/website/src/shared/blocks/blockRenderer.tsx

```diff
@@ -163,6 +162,10 @@ function renderTeaser(key: string, article: PublishedArticle, isPeerArticle = fa
       break
 
     case TeaserType.PeerArticle:
+      if (article.peer?.isDisabled === true) {
```

Comment:
Is there a reason this is not 

```suggestion
      if (article.peer?.isDisabled) {
```